lib/rules/require-blocks-on-newline.js

```diff
@@ -1,16 +1,32 @@
 /**
  * Requires blocks to begin and end with a newline
  *
- * Types: `Boolean` or `Integer`
+ * Types: `Boolean`, `Integer`, `Object`
  *
  * Values: `true` validates all non-empty blocks,
- * `Integer` specifies a minimum number of statements in the block before validating.
+ * `Integer` specifies a minimum number of statements in the block before validating,
+ * `Object` validates all non-empty blocks with comments included in the validation
```

Comment:
Could structure those docs in similar manner - https://github.com/jscs-dev/node-jscs/commit/c536b21ca16b74e44081f4c7d89a38e08439e23c#diff-98bfebb24c1ec81d05c332bf6483e9beR6
?